README.md

```diff
@@ -69,5 +69,26 @@ In that case, you can use the `nuke-testrunner`
 rye run nuke-testrunner -n <path_to_nuke> -t ./tests
 ```
 
+There is also the option to configure your different runners in a `runners.json`.
+This way, you don't need to specify the full nuke path all the time. 
+It's very convenient if you want to test multiple nuke versions in parallel.
+
+The json file is structured like this:
```

Comment:
Could we add besides a template maybe two examples? One simple and one with everything added?

Reply:
I think examples would be better places in a wiki or tutorial. Wdyt?